tests/lib/rules/v-if-else-key.js

```diff
@@ -424,6 +508,62 @@ tester.run('v-if-else-key', rule, {
           line: 6
         }
       ]
+    },
+    {
+      filename: 'test.vue',
+      code: `
+        <template>
+          <div>
+            <ComponentA v-if="foo" />
+
+            <ComponentA v-if="bar" />
+            <ComponentA v-else-if="baz" />
+            <ComponentA v-else />
+          </div>
+        </template>
+        <script>
+        export default {
+          components: {
+            ComponentA
+          }
+        }
+        </script>
+        `,
+      output: `
+        <template>
+          <div>
+            <ComponentA key="component-a-1" v-if="foo" />
+
+            <ComponentA v-if="bar" />
+            <ComponentA key="component-a-3" v-else-if="baz" />
+            <ComponentA key="component-a-4" v-else />
```

Comment:
done, and thanks for pointing that out; I figure that the main benefit of this change is that it will make more sense to the end user?
[0923acc](https://github.com/vuejs/eslint-plugin-vue/pull/2348/commits/0923acc43384a8665b382a56febeeed021520549)
[e978c9d](https://github.com/vuejs/eslint-plugin-vue/pull/2348/commits/e978c9d0e43448cb599dedf4a5168602f80c697c)

Reply:
> does it make a difference which one of those two gets the key?

Probably not; while all but one have a key, the `undefined` key is still unique. But it makes it much more readable.